Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-settings.js

```diff
@@ -107,8 +118,6 @@ angular.module('virtoCommerce.exportModule')
             }
 
             blade.exportDataRequest.providerName = blade.selectedProvider.id;
-            blade.exportDataRequest.dataQuery.skip = undefined;
```

Comment:
Can't these values come from the calling blades (e.g. from the pricing ones)?
I would leave that clearance (or maybe use `delete` as you've done above) to be sure we are not occasionally sending paginated query to the server.